app/src/main/java/tech/vee/veecoldwallet/Activity/ColdWalletActivity.java

```diff
@@ -259,26 +259,30 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
                     switch (txType) {
                         case 2: JsonUtil.checkPaymentTx(activity, jsonMap, accounts);
                                 break;
-                        //case 4: JsonUtil.checkTransferTx(activity, jsonMap, accounts);
-                        //        break;
                         case 3: JsonUtil.checkLeaseTx(activity, jsonMap, accounts);
                                 break;
                         case 4: JsonUtil.checkCancelLeaseTx(activity, jsonMap, accounts);
                     }
                     break;
 
                 case 2:
-                    String seed = QRCodeUtil.parseSeed(qrContents);
-                    if (VEEWallet.validateSeedPhrase(activity, seed)) {
```

Comment:
validateSeedPhrase() must be called to distinguish between normal seed and foreign seed. Here you have removed this logic.